src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/NatureOfControlTypeMapping.java

```diff
@@ -55,59 +85,110 @@ public class NatureOfControlTypeMapping {
 
     OVERSEAS_ENTITIES_FIRM_MAP = new EnumMap<>(NatureOfControlType.class);
     OVERSEAS_ENTITIES_FIRM_MAP.put(OVER_25_PERCENT_OF_SHARES,
-        "OE_OWNERSHIPOFSHARES_MORETHAN25PERCENT_AS_FIRM");
+         "OE_OWNERSHIPOFSHARES_MORETHAN25PERCENT_AS_FIRM");
```

Comment:
Something off with the formatting here I think. Why have the extra spaces been added? On lines 90 and 92 also. I think just the additional indent of 4 spaces was fine.

Reply:
Corrected all the spaces o 4.